### DISCUSSION TITLE
Persisted queries over websocket

### DISCUSSION THREAD

**Discussion by nyvid** [urql-graphql/urql#2988] [Ideas]
Hello guys,
With the subscriptionExchange it is possible to handle all operations over websocket connections. Is it possible or planned to support persisted queries over this exchange too? graphql-ws supports it (https://github.com/enisdenjo/graphql-ws/issues/37#issuecomment-851677998)

**Comment by kitten** [comment]
That's a good point. While I would've asked you to write an RFC, I had some implementation and cleanup notes in mind, so I wrote up an RFC in your stead: https://github.com/urql-graphql/urql/issues/2989

I think this is a pretty overdue feature, so we could do this pretty easily.
